DataFormats/TPC/src/ClusterHardware.cxx

```diff
@@ -0,0 +1,11 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DataFormatsTPC/ClusterHardware.h"
```

Comment:
probably we do not need the cxx files at all, the header file should be self-consistent

Reply:
You are right, I had some problem with the build, and I though it is because of no cxx files at all for that module, but the problem was something else, these files were just left, gonna remove them.